### ISSUE TITLE
Simplified formula compute incorrectly

### ISSUE THREAD

**Issue by JackWH** [cortex-js/compute-engine#76]
Given the formula `((125.5/100)^(0.25))-1` :
- Typing this into a calculator will return the result `0.05842705`
- The LaTeX equivalent of this would be `\left(\left(\frac{125.5}{100}\right)^{0.25}\right)-1`
- Pasting this value into the Cortex demo returns the expected value

However, I'm getting the unexpected result `-0.6652` when simplifying the formula first. Here's a reproducible example:

```js
const formula = '\\left(\\left(\\frac{125.5}{100}\\right)^{0.25}\\right)-1';

const engine = new cortex.ComputeEngine();
const parsed  = engine.parse(formula);

console.info(parsed.N().numericValue);
console.info(parsed.simplify().N().numericValue);
```

The expected result is in the first (non-simplified) line, but after simplification, the second result is very different:

```
0.058427052337 [...]
-0.66529597774 [...]
```

I assumed my LaTeX was wrong, but the MathJSON interpretation of the formula seems accurate (?):

```json
[
  "Subtract",
  ["Power", ["Multiply", ["Rational", 1, 100], 125.5], 0.25],
  1
]
```

Any help would be greatly appreciated, thanks! 🙏

**Comment by JackWH** [comment]
After diving deeper, I now understand that `simplify()` converts the formula into its canonical form. To compare the MathJSON structure between both forms:

```js
// Simplified (canonical):
['Subtract', ['Power', ['Multiply', ['Rational', 1, 100], 125.5], 0.25], 1];

// Unsimplified:
['Subtract', ['Power', ['Delimiter', ['Divide', 125.5, 100]], 0.25], 1];
```

I'm not sure if this is a bug or just an edge case, as I wouldn't expect `simplify()` to change the end result. If I try to "solve"  either MathJSON result by hand, I get the same result — not sure how `Rational` is handled differently by the engine's internals...

Feel free to close if this isn't a bug, just leaving this here for reference otherwise. Thanks!

**Comment by JackWH** [comment]
Thanks, I'm using v0.12.2. The code I'm working with was written over the past 6 months or so, but I only just noticed this so I might've missed a change along the way.

I'm actually not sure how I got the version of the unsimplified expression I posted earlier... I just went to try again and it comes out as you expected. I'd been debugging and trying different combinations of LaTeX at the time, sorry for the confusion!

So we're on the same page, here's a 'final' reproducible example:

```js
const formula = '\\left(\\left(\\frac{125.5}{100}\\right)^{0.25}\\right)-1';
const ce      = new cortex.ComputeEngine({numericPrecision: 14});
const parsed  = ce.parse(formula);

console.info('Simplified = ' + parsed.simplify().N().numericValue);
console.log(parsed.simplify().json);

console.info('Unsimplified = ' + parsed.evaluate().value.numericValue);
console.log(parsed.json);
```

Which outputs like this:

<img width="638" alt="Screenshot 2023-02-14 at 16 40 41@2x" src="https://user-images.githubusercontent.com/627533/218800911-8fd07173-6ba8-4e59-ba83-f7345ea9e5fe.png">
